src/main/java/org/spongepowered/api/world/generation/GenerationRegion.java

```diff
@@ -65,37 +70,74 @@ default Difficulty difficulty() {
         return this.world().difficulty();
     }
 
-    Vector3i centerChunkPos();
+    /**
+     * The minimum chunk co-ordinate of this region.
+     *
+     * @return The chunk co-ordinate as a {@link Vector3i}
+     */
+    Vector3i chunkMin();
```

Comment:
Is this based on the chunk layout of the world (chunk pos vs world block pos)? Maybe we might want to provide the ChunkFormat or whatever that API object is that provides that conversion?

One thing to note is that GenerationRegion inherits the `blockMin()` and `blockMax()` from `Volume` by definition.

Reply:
Fairly sure that this is world co-ordinates, but something to check.